### PR TITLE
Se 1236 add gas consumption results without rules engine integration to the gas report

### PR DESCRIPTION
**Pull request by VoR0220**

### CODE REVIEW

**Review by TJ-Everett — APPROVED** [review]
Looks great! Approved for me

File: docs/userGuides/gasReports/GAS_ESTIMATES_PER_RULE.md [hunk 1]
```diff
@@ -12,102 +12,104 @@ The gas tests were done using simple ERC20 and ERC721 tokens that integrate the
...
+| | safeMint | 30,755 | 152,333 | 208,371 | 56,038 | 36.8% |
+| [Token Max Supply Volatility Rule](../rules/TOKEN-MAX-SUPPLY-VOLATILITY.md) | transferFrom | 33,977 | 87,027 | n/a | 0 | 0 |
+| | burn | 10,981 | 44,409 | 50,763 | 6,354 | 14.3% |
+| | safeMint | 30,755 | 152,333 | 178,445 | 26,112 | 17.1% |
+| [Token Max Trading Volume](../rules/TOKEN-MAX-TRADING-VOLUME.md) | transferFrom | 33,977 | 87,027 | 92,392 | 5,365 | 6.2% |
+| | burn | 10,981 | 44,409 | 50,763 | 6,354 | 14.3% |
+| | safeMint | 30,755 | 152,333 | 157,698 | 5,365 | 3.5% |
+| [Token Minimum Hold Time](../rules/TOKEN-MIN-HOLD-TIME.md) | transferFrom | 33,977 | 87,027 | 90,329 | 3,302 | 3.8% | 
+| | burn | 10,981 | 44,409 | 47,416 | 3,007 | 6.8% |
+| | safeMint | 30,755 | 152,333 | n/a | n/a | n/a |
```
Comment (ShaneDuncan602) [inline docs/userGuides/gasReports/GAS_ESTIMATES_PER_RULE.md]: I don't think they wanted the base numbers added to the table. They just wanted those numbers added to the blurb at the top.

**Review by mpetersoCode55 — APPROVED** [review]

File: docs/userGuides/gasReports/GAS_ESTIMATES_PER_RULE.md (hunk 1 above)
Comment (VoR0220) [reply docs/userGuides/gasReports/GAS_ESTIMATES_PER_RULE.md]: Can fix this.

File: test/util/gasReport/GasReportRegularTokens.t.sol [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    function test_ERC721_Transfer() public {
+        erc721Mock.mint(address(this), 1);
+        startMeasuringGas("ERC721_Transfer");
+        erc721Mock.transferFrom(address(this), address(1), 1);
+        uint256 gasDelta = stopMeasuringGas();
+        console.log("Gas Delta: %s", gasDelta);
+    }
+}
```
Comment (ShaneDuncan602) [inline test/util/gasReport/GasReportRegularTokens.t.sol]: Can you change the log statements on these to reflect the token and action...ERC721 mint Baseline:, etc.

Comment (VoR0220) [reply test/util/gasReport/GasReportRegularTokens.t.sol]: yep.

**Review by ShaneDuncan602 — APPROVED** [review]
Looks great

**Pull request merged**